hello.go

```diff
@@ -86,13 +86,17 @@ func main() {
 
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		tmpl.Execute(w, data)
+		log.Printf("%s request from %s for %s", r.Method, r.Host, r.URL)
 	})
 
+	fs := http.FileServer(http.Dir("./assets"))
+	http.Handle("/assets/", http.StripPrefix("/assets/", fs))
+
 	port := os.Getenv("PORT")
 	if port == "" {
 		port = "8080"
 	}
 
-	log.Print("Hello from Cloud Run! The container started successfully and is listening for HTTP requests on $PORT.")
+	log.Printf("Hello from Cloud Run! The container started successfully and is listening for HTTP requests on $PORT, %s.", port)
```

Comment:
The goal of this sentence is to teach to listen on $PORT, and not to hardcode a number.
So adding a number might add confusion.
Maybe, only display the actual number if $PORT is unset? (e.g. for local development)

Reply:
Reverted.